python/foglamp/plugins/north/ocs/ocs.py

```diff
@@ -103,7 +103,7 @@
     "URL": {
         "description": "The URL of OCS (OSIsoft Cloud Services) ",
         "type": "string",
-        "default": "https://dat-a.osisoft.com/api/omf",
+        "default": "https://dat-a.osisoft.com/api/tenants/TENANT_ID_PLACEHOLDER/namespaces/NAMESPACE_ID_PLACEHOLDER/omf",
```

Comment:
I wonder if we might be better having TENANT_ID and NAMESPACE_ID as separate configuration items and construct the URL rather than have a URL like this we placeholders.